src/samples/ChildBenefits/index.tsx

```diff
@@ -0,0 +1,276 @@
+/* eslint-disable react/button-has-type */
```

Comment:
Can we rename the folder samples to something more appropriate like app or applications

Reply:
samples is the default name from the Pegasystems SDK fork, would React still pick up the folder if it was named something else? This FAQ suggests it can be named anything https://reactjs.org/docs/faq-structure.html

What shall we go with?

Reply:
yes it could be named anything, but its a best practice to have meaningful names or name that states purpose, module. feature etc
Only the import in index.tsx for TopLevelApp that has this folder name need to be updated I believe

Reply:
Any preference on using the name /Applications or /Journeys ?

Am I right in thinking that ChildBenefitClaim is a Journey of ChildBenefits?

Reply:
Have gone with /journeys.